robot/basestation/static/js/elements/feed/cmd_input.js

```diff
@@ -0,0 +1,46 @@
+$(document).ready(() => {
+  const send_cmd = $('#send-serial-cmd')
+  const cmd_input = $('#serial-cmd-input')
+
+  function handleCMD() {
+    const full_cmd = cmd_input.val()
+    const prefix = full_cmd.split(' ').shift()
+    const cmd = full_cmd.split(' ').pop()
+    if (cmd != '' && cmd != full_cmd) {
+      sendCMD(prefix, cmd)
+    }
```

Comment:
Role of this function should be to return a boolean if the command is valid or not. Right now, its doing multiple things at once

Reply:
Alright, I split this into two functions called `isValidCMD`, and `getCMD`. In order separate these roles into their own respective functions